CONTRIBUTING.md

```
@@ -54,11 +53,11 @@ Before you submit your Pull Request (PR) consider the following guidelines:
 
 6. Make your changes, **including test cases and documentation updates where appropriate**.
 
-7. Follow our [coding rules](#rules).
+7. Follow our [coding rules](#coding-rules).
 
 8. Run all tests and checks locally, as described in the [development guide][developing], and ensure they pass. This saves CI hours and ensures you only commit clean code.
```

markdownlint-cli2 (0.17.2)

Reference links and images should use a label that is defined
Missing link or image reference definition: "developing"

(MD052, reference-links-images)